1832_check_if_it_is_a_pangram.py

```diff
@@ -0,0 +1,6 @@
+class Solution:
+    def checkIfPangram(self, sentence: str) -> bool:
+        checker = [0] * 26
+        for i in sentence:
```

Comment:
"i" is used for index, so in this case it's better to use "character" or "letter".